### PR TITLE
Add SBD advice from archived TIDs

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Added various advice for using SBD.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1202661
* jsc#DOCTEAM-725

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP5
  - [x] 15 SP4
  - [ ] 15 SP3
  - [ ] 15 SP2
  - [ ] 15 SP1
- SLE-HA 12
  - [ ] 12 SP5
  - [ ] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/ha_storage_protection.xml [hunk 1]
```diff
@@ -706,9 +729,16 @@ Timeout (msgwait)  : 10
...
-    <para>Restart the cluster services on each node:</para>
-    <screen>&prompt.root;<command>crm cluster restart</command></screen>
+    <para>Restart the cluster services on all nodes at once by using the <option>--all</option>
+     option:</para>
+    <screen>&prompt.root;<command>crm cluster restart --all</command></screen>
     <para> This automatically triggers the start of the SBD daemon. </para>
+    <important>
+     <title>Restart cluster services for SBD changes</title>
+     <para>
+       If any SBD metadata changes, you must restart the cluster services again.
```
Comment (zzhou1) [inline xml/ha_storage_protection.xml:739]: Also good to mention that, in a running cluster, the customer may want to run ` crm maintenance on` to put the cluster in the maintenance mode let the application running while restart the cluster stack. And, `crm maintenance off` after that.

File: xml/ha_storage_protection.xml [hunk 2]
```diff
@@ -969,9 +999,16 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
...
-     <screen>&prompt.root;<command>crm cluster restart</command></screen>
-     <para> This automatically triggers the start of the SBD daemon. </para>
+     <para>Restart the cluster services on all nodes at once by using the <option>--all</option>
+     option:</para>
+    <screen>&prompt.root;<command>crm cluster restart --all</command></screen>
+    <para> This automatically triggers the start of the SBD daemon. </para>
+    <important>
+     <title>Restart cluster services for SBD changes</title>
+     <para>
+       If any SBD metadata changes, you must restart the cluster services again.
```
Comment (zzhou1) [inline xml/ha_storage_protection.xml:1009]: same comment as the previous one.

File: xml/ha_storage_protection.xml [hunk 3]
```diff
@@ -216,6 +216,18 @@
     <para>An SBD device can be shared between different clusters, as
      long as no more than 255 nodes share the device. </para>
    </listitem>
+   <listitem>
+     <para>
+       When using more than one SBD device, all nodes must have a slot in all SBD devices.
+       Fencing does not work with an asymmetric SBD setup.
```
Comment (zzhou1) [inline xml/ha_storage_protection.xml:222]: Out of my head, I just came up with a small bash scrpt for users, not sure if appropriate as for DOC per say. 

``` bash
SBD_DEVICE="/dev/disk/by-partlabel/sbd-155;/dev/disk/by-partlabel/sbd-155-1"
crm cluster run " for i in `echo $SBD_DEVICE|tr ';' ' ' `; do sbd -d $i list; done "
```

Similarly, the sbd "dump" command to verify timeouts

``` bash
SBD_DEVICE="/dev/disk/by-partlabel/sbd-155;/dev/disk/by-partlabel/sbd-155-1"
crm cluster run " for i in `echo $SBD_DEVICE|tr ';' ' ' `; do sbd -d $i dump; done "
```

File: xml/ha_storage_protection.xml (hunk 1 above)
Comment (gao-yan) [reply xml/ha_storage_protection.xml:739]: This chapter thoroughly covers probably all about "maintenance tasks":

https://susedoc.github.io/doc-sleha/main/html/SLE-HA-administration/cha-ha-maintenance.html

So a reference here probably would be good enough.

**Review by gao-yan — COMMENTED** [review]
Thanks for taking care of this, @tahliar !

File: xml/ha_storage_protection.xml (hunk 1 above)
Comment (tahliar) [reply xml/ha_storage_protection.xml:739]: Thanks Roger and Yan, I'll add a reference to the maintenance chapter.

**Review by zzhou1 — APPROVED** [review]

**Review by dariavladykina — APPROVED** [review]
LGTM.

**Pull request merged**